commit f39da64fb3ce370e0f3b24d8d96a9c5e6ecd0a79 (refs/changes/24/5624/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-22T16:08:01+00:00 (8 months ago)
    
    TRITON-1248 hermes should .gitignore more build artifacts
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/.gitignore b/.gitignore
index e2b4aef..dd0a1cc 100644
--- a/.gitignore
+++ b/.gitignore
@@ -4,4 +4,6 @@ bin/node
 node/
 proto/
 downloads/
+/build
+/make_stamps
 .*.sw?
